elastic/kibana
# [Security solution] 8.8.1 GenAi Connector Feature Flag

## Summary

Accidentally [auto-merged an 8.9 feature](https://github.com/elastic/kibana/pulls/157288) that we are putting in 8.8.1 for an evaluation. I meant to put a feature flag on it first. This is the feature flag. Sorry for the early merge!

### To test:

1. Checkout branch and run
2. Try to create a GenAI connector, note that it is not present in the connector list by default
3. Add the feature flag to dev yml `xpack.stackConnectors.enableExperimental: ['genAiEnabled']`
4. Refresh and try to create a GenAI connector, you will now be able to do so

### Caveat:

If you create a GenAI connector and then disable the feature flag, it will show as a disabled connector. I think we should let this be, because to resolve we'd need to add a feature flag for the triggersActionsUi plugin and that would be 3 feature flags to turn on GenAI. If y'all are ok with the disabled connector, i think I should just leave it.
<img width="1728" alt="Screenshot 2023-06-01 at 8 39 54 PM" src="https://github.com/elastic/kibana/assets/6935300/1389c177-c95b-4251-b3c3-52a6797488e1">
